src/Confluent.Kafka/Producer.cs

```diff
@@ -241,19 +257,19 @@ public void HandleDeliveryReport(Message message)
         ///     Initializes a new Producer instance.
         /// </summary>
         /// <param name="config">
-        ///     librdkafka configuration parameters (refer to https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md)
-        ///     TODO: Link to confluent-kafka-dotnet page with dotnet specific parameters also (i.e. default.topic.config).
+        ///     librdkafka configuration parameters (refer to https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md).
+        ///     Topic configuration parameters are specified via the "default.topic.config" sub-dictionary config parameter.
         /// </param>
         /// <param name="manualPoll">
         ///     If true, does not start a dedicated polling thread to trigger events or receive delivery reports -
-        ///     you must call the Poll method periodically instead.
+        ///     you must call the Poll method periodically instead. Typically you should set this parameter to false.
         /// </param>
         /// <param name="disableDeliveryReports">
-        ///     If true, disables notification of delivery reports. Note: if set to true and you use a ProduceAsync variant that return
-        ///     a Task, the Tasks will never complete. Generally you should leave this parameter as false. Set it to true for "fire and
+        ///     If true, disables delivery report notification. Note: if set to true and you use a ProduceAsync variant that returns
+        ///     a Task, the Tasks will never complete. Typically you should set this parameter to false. Set it to true for "fire and
```

Comment:
oh, i remember. this isn't exposed on `Producer<TKey,TValue>`, only `Producer`. And the entire `Producer` class is marked as unstable - though at this point i'm in favour of keeping it. It exposes two advanced use case things, yet keeps them out of the interface every one is going to use in practice.